src/utils/logger.js

```diff
@@ -1,13 +1,33 @@
-const noop = () => {};
+/* eslint-disable no-confusing-arrow */
+let debug = __DEV__;
 
-export const log = __DEV__ ? (...args) => console.log(
-  `%c ${args.join(' ')}`, 'background: #8cdcf2; padding: 3px 0; color: black;',
-) : noop;
+const prepareArgs = args => args.map(
+  value => typeof value === 'string' ? value : JSON.stringify(value),
+).join(' ');
 
-export const error = __DEV__ ? (...args) => console.log(
-  `%c ${args.join(' ')}`, 'background: red; padding: 3px 0; color: white;',
-) : noop;
+export const error = (...args) => {
```

Comment:
Using an object instead of explicit named exports should work:

```javascript
const log = {}
const levels = [
	{ level: 'debug', background: '#FFF', color: '#000' },
	{ level: 'info', background: '#FFF', color: '#000' },
	{ level: 'warn', background: '#FFF', color: '#000' },
	{ level: 'error', background: '#FFF', color: '#000' },
]

const bindLevels = (enabled = true, logLevel = levels.length) => {
	levels.forEach(({ level, background, color }, levelIndex) => {
		let logFn = noop
		if (enabled === true && logLevel < levelIndex) {
			logFn = (...args) => {
    			    console.log(`%c ${prepareArgs(args)}`, `background: ${background}; padding: 3px 0; color: ${color};`);
 			}
		}

		log[level] = logFn
	})
}

export const enable = () => { bindLevels() }

export const disable = () => { bindLevels(false) }

export const setLogLevel = logLevel => { bindLevels(true, logLevel) }

enable()

export default log
```